subsys/logging/backends/Kconfig.net

```diff
@@ -15,6 +15,15 @@ config LOG_BACKEND_NET
 
 if LOG_BACKEND_NET
 
+config LOG_BACKEND_NET_SERVER_RUNTIME
```

Comment:
I was using it mostly for the sample app so that it was easier and more obvious to test and show future users what where the various capabilities of the `syslog_net` backend. But I can remove it no problem.